### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by liu-song-6**
small fixes in examples/cpp/TCPSendStack.cc and tools/profile.py

### CODE REVIEW

File: examples/cpp/TCPSendStack.cc [hunk 1]
```diff
@@ -27,7 +27,7 @@ struct stack_key_t {
   int kernel_stack;
 };
 
-BPF_STACK_TRACE(stack_traces, 10240)
+BPF_STACK_TRACE(stack_traces, 10240);
```
Comment (yonghong-song) [inline examples/cpp/TCPSendStack.cc]: The `BPF_STACK_TRACE` defines like:
```
#define BPF_STACK_TRACE(_name, _max_entries) \
  BPF_TABLE("stacktrace", int, struct bpf_stacktrace, _name, _max_entries);
```
There is already a `;` at the end.

I do recognize the usage is inconsistent (e.g., at tools directory below)
```
-bash-4.2$ egrep -r BPF_STACK_TRACE
deadlock_detector.c:BPF_STACK_TRACE(stack_traces, 655360);
memleak.py:BPF_STACK_TRACE(stack_traces, 10240)
offcputime.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
offwaketime.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
old/profile.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
profile.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
stacksnoop.lua:BPF_STACK_TRACE(stack_traces, 128)
trace.py:                stack_table = "BPF_STACK_TRACE(%s, 1024);" % self.stacks_name \
wakeuptime.py:BPF_STACK_TRACE(stack_traces, STACK_STORAGE_SIZE)
stackcount.py:BPF_STACK_TRACE(stack_traces, 1024);
```

Ideally, we may want to change macro instead to force user to add `;` just like `BPF_TABLE`.  This risks breaking some existing codes. Any other concerns?

Comment (liu-song-6) [reply examples/cpp/TCPSendStack.cc]: I see. I "fixed" the macro in my local branch. This does make things tricky. Maybe we just keep this part as is.

**Comment by yonghong-song** [comment]
@liu-song-6 Could you make a change to BPF_STACK_TRACE macro by removing the ";" in the macro definition? This will make BPF_STACK_TRACE macro consistent with other macros, e.g., BPF_HASH, BPF_HISTOGRAM, etc. Please search through the repository to fix the usage of macros as well. Thanks!

**Comment by liu-song-6** [comment]
@yonghong-song The new version covers all the instances of BPF_STACK_TRACE().

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

**Pull request merged**